packages/turf-mask/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@turf/mask",
+  "version": "3.10.3",
+  "description": "Takes a Polygon and an optional mask and returns the outside area.",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "scripts": {
+    "test": "node test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "turf",
+    "mask",
+    "polygon"
+  ],
+  "author": "Turf Authors",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Turfjs/turf/issues"
+  },
+  "homepage": "https://github.com/Turfjs/turf",
+  "dependencies": {},
+  "devDependencies": {
+    "benchmark": "^1.0.0",
```

Comment:
Use benchmark `2.x`, also I've got an easy example that you could almost copy paste and only have to replace a few things.

https://github.com/Turfjs/turf/blob/master/packages/turf-line-intersect/bench.js